index.html

```diff
@@ -290,6 +290,18 @@ <h2>About me</h2>
         </div>
       </div>
     </main>
+    
+    <div class="footer">
+      <p>
```

Comment:
- [ ] Using a `footer` tag here is better than the generic `div` tag. This is to ensure best semantic practices.